opentracing-mongo-common/src/main/java/io/opentracing/contrib/mongo/common/TracingCommandListener.java

```diff
@@ -172,10 +172,14 @@ Span buildSpan(CommandStartedEvent event) {
         .buildSpan(mongoSpanNameProvider.generateName(event))
         .withTag(Tags.SPAN_KIND.getKey(), Tags.SPAN_KIND_CLIENT);
 
-    Span span = spanBuilder.start();
-    for (SpanDecorator decorator : decorators) {
-      decorator.commandStarted(event, span);
-    }
+        if (tracer.activeSpan() != null) {
```

Comment:
there is no need explicitly add active span as parent.
it is done automatically.
you can verify that if you run your tests without this change.

Reply:
Yes seems to work even without the change. Thanks!